README.md

```diff
@@ -9,13 +9,46 @@ This project is meant to demonstrate key features of SilverStripe CMS. It combin
 This project is used to build the demonstration environment when a user request a SilverStripe demo from [SilverStripe.com](https://silverstripe.com). 
 
 It's not meant to be used directly by the community. However, there's nothing stopping third party from re-using it for their own demo. SilverStripe doesn't provide official support for this project.
-    
 
-## Docker setup
+## Local Development
 
-WARNING: The docker image is specifically crafted for our internal Kubernetes stack. It is not supposed to suit any other needs.
+This project is aimed to be deployed in an internal Kubernetes stack,
+as a set of Docker containers. You can approximate this setup locally
+by running it through [docker-compose](https://docs.docker.com/compose/).
+
+```
+cd docker
+docker-compose up
+```  
+
+Once all containers have launched, you should be able to access the site
+under `http://localhost:8200`.
+
+If you need access to individual containers (e.g. SSH into the web container),
+you can use `docker-compose exec` to get an interactive shell:
+
+```
+docker-compose exec web /bin/bash
+``` 
+
+Note that you should run commands as the web user, through `gosu www-data`.
```

Comment:
could do `docker-compose exec -u www-data web /bin/bash` instead

Reply:
Hah I tried that, but didn't work. I think I did `docker exec` rather than `docker-compose exec`.